src/wasm-interpreter.h

```diff
@@ -1375,6 +1375,10 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     NOTE_EVAL1(value);
     return Literal(value.geti31(curr->signed_));
   }
+  Flow visitCallRef(CallRef* curr) {
+    NOTE_ENTER("CallRef");
+    WASM_UNREACHABLE("TODO (gc): ref.call");
+  }
```

Comment:
It looks like this should be up with the other nonconstant expressions with body `{ WASM_UNREACHABLE("unimp"); }`.